information-schema/information-schema-slow-query.md

```diff
@@ -7,6 +7,14 @@ summary: Learn the `SLOW_QUERY` information_schema table.
 
 The `SLOW_QUERY` table provides the slow query information of the current node, which is the parsing result of the TiDB slow log file. The column names in the table are corresponding to the field names in the slow log.
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, the `Slow Query Table` is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```

Comment:
```suggestion
> Currently, the `SLOW_QUERY` table is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```

Reply:
If we don't plan to support this table in near future for Serverless Tier, we could remove "Currently".